CorsixTH/Lua/endconditions.lua

```diff
@@ -32,13 +32,15 @@ local local_criteria_variable = {
     -- New criteria
   {name = "staff_happiness",  icon = 16, icon_file = "MPointer", formats = 2, two_tooltips = true},
   {name = "patient_happiness",icon = 18, icon_file = "MPointer", formats = 2, two_tooltips = true},
+  {name = "year",             icon = 58, icon_file = "Font04V",  formats = 4, two_tooltips = true}, -- Starts from 0 not 2000
```

Comment:
If you rename to "years_played" (or something like that), you avoid the ambiguity between 0 and 2000.

Reply:
Another option here is to add the base year in `Date`, and do everything in the absolute year. That may need to subtract the base year again in some computations, not sure.